captum/attr/_core/feature_permutation.py

```diff
@@ -40,8 +50,127 @@ def attribute(  # type: ignore
         additional_forward_args: Any = None,
         feature_mask: Optional[TensorOrTupleOfTensors] = None,
         ablations_per_eval: int = 1,
-        **kwargs: Any
+        **kwargs: Any,
     ) -> TensorOrTupleOfTensors:
+        r"""
+        This attribution method essentially implements the permutation feature
+        importance algorithm, as described here:
+        https://christophm.github.io/interpretable-ml-book/feature-importance.html
+
+        A basic tl;dr of the algorithm is:
+
+        perm_feature_importance(batch):
+            importance = dict()
+            baseline_error = error_metric(model(batch), batch_labels)
+            for each feature:
+                permute this feature across the batch
+                error = error_metric(model(permuted_batch), batch_labels)
+                importance[feature] = baseline_error - error
+                "un-permute" the feature across the batch
+
+            sort importance in descending order
```

Comment:
nit: do we actually perform sorting in our implementation ?

Reply:
Good point - no